.github/workflows/auto-update-otel-sdk.yml

```diff
@@ -0,0 +1,52 @@
+name: Auto-update OpenTelemetry SDK
+
+on:
+  schedule:
+    # hourly at minute 46
+    - cron: "46 * * * *"
+  workflow_dispatch:
+
+jobs:
+  auto-update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Check latest version
+        run: |
+          latest_version=$(gh release view \
```

Comment:
`latest_version` and `current_version` don't seem to be used in this file

Reply:
sorry, not sure why I pushed such an early draft